google/cloud/bigtable/admin_client.cc

```diff
@@ -15,6 +15,7 @@
 #include "google/cloud/bigtable/admin_client.h"
 #include "google/cloud/bigtable/internal/common_client.h"
 
+class CommonClient;
```

Comment:
Is this really in the global namespace? That's not good for our users.

If it's not, shouldn't it be forward-declared in the correct namespace?

Reply:
Removed, I don't know why I put that there (or how).